Our-Project.ipynb

```diff
@@ -7,6 +7,111 @@
     "# This is the start!!!"
```

Comment:
<p>Lets make this a separate function. Shall we call it - <code>get_top_common_genres</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-ML-AY2021S1-0037/CS3244-ML/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Nice stuff. Couple of comments in ReviewNB. I feel we could push the pre-processed changes faster than the KNN changes. It doesnt really affect anyone apart from the changes to `common_genres`. That being, and this is up to your discretion, perhaps you could shift those KNN changes to another branch? Then we could push the pre-process changes earlier.

Reply:
ok i have branched out the kNN part

Reply:
sweet